pennylane/gradients/finite_difference.py

```diff
@@ -250,9 +250,9 @@ def _finite_diff_new(
             device shots for the new return types output system.
 
     Returns:
-        tensor_like or tuple[list[QuantumTape], function]:
+        tensor_like or tuple[tensor_like] or tuple[tuple[tensor_like]] or tuple[list[QuantumTape], function]:
 
-        - If the input is a QNode, a tensor
+        - If the input is a QNode, a tensor, tuple or tuple of tuple
           representing the output Jacobian matrix of size ``(number_outputs, number_gate_parameters)``
           is returned.

```

Comment:
@rmoyard what do you think?

Great catching that we need to update this!
```suggestion
        - If the input is a QNode, an object representing the output Jacobian matrix
          of size ``(number_outputs, number_gate_parameters)`` is returned. The
          type of the object returned is either a tensor, a tuple or a nested tuple depending
          on the nesting structure of the output.
```